client/components/Help/Help.js

```diff
@@ -68,4 +107,15 @@ const mapStateToProps = ({ problems }) => {
     };
 };
 
-export default connect(mapStateToProps)(Help);
+const mapDispatchToProps = (dispatch) => {
```

Comment:
usually you will simply pass an object instead of a function and connect will AUTOMATICALLY wrap that in dispatch:

export default connect(mapStateToProps, {
    getProblems,
    addProblem
 })(Help);